src/folio/users-api.ts

```diff
@@ -34,11 +34,28 @@ export default class UsersAPI extends FolioAPI {
     return blocks.manualblocks
   }
 
+  async getFeeFineActions(accountId) {
+    const urlParams = this.buildCqlQuery({ accountId: accountId })
+    const actions = await this.get(`/feefineactions`, { params: urlParams })
+    return actions.feefineactions
+  }
+
+
   async getAccounts(userId: string): Promise<any> {
     const urlParams = this.buildCqlQuery({ userId: userId })
-    const accounts = await this.get<AccountsResponse>(`/accounts`, { params: urlParams })
+    const response = await this.get<AccountsResponse>(`/accounts`, { params: urlParams })
 
-    return accounts.accounts
+    const accountsWithActions = await Promise.all(
+      response.accounts.map(async (account) => {
+        const actions = await this.getFeeFineActions(account.id);
+        return {
+          ...account,
+          actions: actions,
+        };
+      })
+    );
+  
+    return accountsWithActions;   
   }
```

Comment:
is there a difference between eagerly adding the actions here, for every account, vs. adding this resolver to `index.ts` like:
```ts
  Account: {
    user({ userId }, args, { dataSources: { users } }, info) {
      return users.getUser(userId)
    },
    ...
    feeFine({ feeFineId }, args, { dataSources: { feefines } }, info) {
      return feefines.getFeeFine(feeFineId)
    },
    feeFineActions({ id }, args, { dataSources: { users } }, info) {
      return users.getFeeFineActions(id)
    },
  },
```
I might be missing some pieces here, but I'd think that'd allow you to avoid the call to to `/feefineactions` unless the query explicitly included it